cmd/juju/status/history.go

```diff
@@ -47,13 +48,13 @@ This command will report the history of status changes for
 a given entity.
 The statuses are available for the following types.
 -type supports:
-    juju-unit: will show statuses for the unit's juju agent.
-    workload: will show statuses for the unit's workload.
-    unit: will show workload and juju agent combined for the specified unit.
+    container: will show statuses for containers.
+    juju-container: will show statuses for the container's juju agent.
```

Comment:
all of these repeating "will show statuses" makes me think we have a refactoring problem with the doc text. :)
Maybe:
The statuses can be shown for a variety of types. We support:
  - container: only containers and not their host machines
  - juju-container: the status messages from the agent that is managing containers
  - machine: statuses that occur due to provisioning of a machine
  - juju-machine: status of the agent that is managing a machine

I'm not sure of all of those bits of text, I'm happy to brainstorm them with you. But remove some of the redundancy and also give a little bit more of actual *why* you would care to see the messages for any given one.

Things like: "machine: will show statuses for machines" seems both redundant, verbose, and completely useless.